lib/editor-linter.coffee

```diff
@@ -77,7 +91,11 @@ class EditorLinter
 
   # This method sets or gets the lock status of given type
   _lock: (wasTriggeredOnChange, value) ->
```

Comment:
Yep, but there's a lot of them, so will do them after we're done bug fixing and have plenty of time.